labelbox/schema/global_key.py

```diff
@@ -0,0 +1,15 @@
+from pydantic import BaseModel
+
+from labelbox.utils import camel_case
+
+
+class _CamelCaseMixin(BaseModel):
+
+    class Config:
+        allow_population_by_field_name = True
+        alias_generator = camel_case
+
+
+class AssignGlobalKeyToDataRowInput(_CamelCaseMixin):
```

Comment:
Discussed with @msokoloff1 offline on this. Let's use a dict as the parameter, for a few reasons:
- Most other places in code use primitives for parameters, so we want to follow the pattern for consistency
- Customers would need to have an extra line of import to use the assign function, which is moving away from our goal of improving usability
- Customers like dicts better as input params

We should remove this type, and `assign_global_keys_to_data_rows` should take in a dict, and validate the input.